modules/install-sno-installing-with-the-assisted-installer.adoc

```diff
@@ -8,18 +8,22 @@
 
 Use the Assisted Installer to install the single-node cluster.
 
+.Prerequisites
+
+* Ensure that the boot drive order in the server BIOS settings defaults to booting the server from the target installation disk.
+
 .Procedure
 
-. Attach the {op-system} discovery ISO to the target host.
+. Attach the {op-system-first} discovery ISO image to the target host.
 
-. Configure the boot drive order in the server BIOS settings to boot from the attached discovery ISO and then reboot the server.
+. Configure the server to boot from the {op-system} discovery ISO image. The discovery ISO image writes the {op-system} system configuration to the target installation disk and automatically triggers a server restart.
```

Comment:
I would avoid using the word "configure" here because it might incentivize the user to undo the "boot-to-disk-by-default" step we asked them to do in the first place.

I would just say: `Boot the server from the discovery ISO image.` Also, while the ISO image does run RHCOS, I think including that is more confusing than it is helpful.